lib/ach_client/providers/sftp/ach_status_checker.rb

```diff
@@ -47,6 +47,7 @@ def self.in_range(start_date:, end_date:)
         if ach.addenda.length == 0
           # If there are no addenda, it is a success.
           AchClient::SettledAchResponse.new(
+            amount: (ach.amount / 100.0).to_d,
             date: batch.header.effective_entry_date
```

Comment:
Correct - that is the case for banks due to NACHA spec, but not for the API providers